packages/core/test/transient.test.ts

```diff
@@ -255,63 +255,6 @@ describe('transient states (eventless transitions)', () => {
     expect(actorRef.getSnapshot().value).toEqual({ A: 'A4', B: 'B4' });
   });
 
-  it('should execute all eventless transitions in the same microstep (with `always`)', () => {
```

Comment:
What was wrong with these deleted tests?

Reply:
Those were copies of other test nearby. In the past we had 2 copies of those since one was using `''` and the other one was using `always` but now both were using the latter